api/src/certification/results/infrastructure/serializers/certificate-serializer.js

```diff
@@ -59,6 +60,7 @@ const serialize = function ({ certificate, translate }) {
       globalLevelLabel: certificate.globalLevel.getLevelLabel(translate),
       globalSummaryLabel: certificate.globalLevel.getSummaryLabel(translate),
       globalDescriptionLabel: certificate.globalLevel.getDescriptionLabel(translate),
+      meshLevel: certificate.globalLevel.meshLevel.split('_').at(-1),
```

Comment:
Suggestion: l'extraire dans une fonction afin de comprendre directement de quoi il s'agit.

```suggestion
      meshLevel: _getLevel(certificate),
      
      
      // Ailleurs dans la page
      _getLevel(certificate) {
           return certificate.globalLevel.meshLevel.split('_').at(-1)
      }
```